src/chat/participant.ts

```diff
@@ -259,23 +265,23 @@ function filterContextHistory(
   if (filteredHistory.length === 0) {
     return [];
   }
-  const messages: LanguageModelChatMessage[] = [];
+  let messages: string = "These are the previous messages in the conversation:\n\n";
   for (const turn of filteredHistory) {
     // don't re-use previous prompts since the model may misinterpret them as part of the current prompt
     if (turn instanceof ChatRequestTurn) {
       if (turn.participant === PARTICIPANT_ID) {
-        messages.push(userMessage(turn.prompt));
+        messages = `${messages}\n\nUSER: "${turn.prompt}"`;
```

Comment:
formatting witchery 🎩 --- basically, adding this clarifies the prompt for the consumption of the AI model. Not sure why, though.